pkgs/servers/web-apps/pixelfed/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, php
+, pkgs
+, nixosTests
+, dataDir ? "/var/lib/pixelfed"
+, runtimeDir ? "/run/pixelfed"
+}:
+
+let
+  package = (import ./composition.nix {
+    inherit pkgs;
+    inherit (stdenv.hostPlatform) system;
+    noDev = true; # Disable development dependencies
+  }).overrideAttrs (attrs : {
+    installPhase = attrs.installPhase + ''
+      rm -R $out/bootstrap/cache
+      # Move static contents for the NixOS module to pick it up, if needed.
+      mv $out/bootstrap $out/bootstrap-static
+      mv $out/storage $out/storage-static
+      ln -s ${dataDir}/.env $out/.env
+      ln -s ${dataDir}/storage $out/
+      ln -s ${dataDir}/storage/app/public $out/public/storage
+      ln -s ${runtimeDir} $out/bootstrap
```

Comment:
This is a reasonable approach, though I recall that if you _really_ dig deep into `laravel` internals there is a way to avoid this. Probably not worth the effort, though...

Reply:
Yeah, I would love to clean it up later though! :)